### PR TITLE
Arc-Luminol

### PR DESCRIPTION
**Pull request by KayeArray**
:cl: banthebantz
add: Adds a memechem to honour our dearly departed Arclumin
add: Known as Arc-Luminol this mysterious and dangerous substance has baffled scientists with its highly unique and unstable properties, make it at your own risk!
add: Made with 2 parts Teslium 2 parts Colorful Reagent 2 parts Liquid Dark Matter 2 parts Rotatium along with a plasma catalyst and 400k heat with a yield of 50%.

/:cl:

[]: Here is a funny play on words memechem that is rather dangerous to both the user and anyone around them, it's not super lethal but it sure is pretty crazy and !FUN!

### CODE REVIEW

File: code/game/objects/effects/decals/Cleanable/misc.dm [hunk 1]
```diff
@@ -43,6 +43,23 @@
...
+	name = "charged goo"
+	desc = "It's sparking and dangerous!."
+	luminosity = 1
+	icon_state = "chargedgoop"
+
+/obj/effect/decal/cleanable/arc/Crossed(atom/movable/O)
+	if(ishuman(O))
+		var/mob/living/carbon/human/H = O
+		if(H.shoes)
+			H.electrocute_act(5, 1, 1, stun = FALSE)
```
Comment (ghost) [inline code/game/objects/effects/decals/Cleanable/misc.dm:56]: Just change "stun = FALSE" to "FALSE"

Comment (ghost) [reply code/game/objects/effects/decals/Cleanable/misc.dm:56]: wait shit, hold on

File: code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm [hunk 2]
```diff
@@ -277,4 +277,70 @@
...
 
 		if(reac_volume >= 6)
-			T.freon_gas_act() //freon in my pocket
\ No newline at end of file
+			T.freon_gas_act() //freon in my pocket
+
+/datum/reagent/arclumin//memechem made in honour of the late arclumin
+	name = "Arc-Luminol"
+	id = "arclumin"
+	description = "You have no idea what the fuck this is but it looks absurdly unstable, it emits a healthy glow so ingestion is probably not a good idea."
```
Comment (ghost) [inline code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm]: "healthy" is kinda not right?? "radiant" might be a better word to use?
Change the comma to a period.
Change "it emits" to "It is emitting"

Comment (KayeArray) [reply code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm]: Eh it can be used but now I don't like it so CHANGED

**Comment by KayeArray** [comment]
i like memes

**Review by j-mie — APPROVED** [review]

**Pull request merged**